CHANGES.rst

```diff
@@ -4,6 +4,13 @@ Latest changes
 Development
 -----------
 
+Gael Varoquaux
+
+    Avoid oversubscription when there are multiple nested parallel loops.
+    As a result the system avoids fork bombs with recursive parallel
```

Comment:
Let's be more specific about this change:

    the system avoids thread bombs with ....

This change is not about the "fork" system call or new process creation.